maven-reactor-projects/maven-reactor-projects.groovy

```diff
@@ -0,0 +1,11 @@
+/**
+ * This Groovy script adds a custom value for each Maven project in the reactor.
+ */
+
+def buildScan = session.lookup('com.gradle.maven.extension.api.scan.BuildScanApi')
+
+buildScan.executeOnce('maven-reactor-projects') { api ->
```

Comment:
❌ This is only needed for things that are global. If you want something per project, you can just access the project directly, since this script is already executed for each project.

Reply:
Isn't `session.projects` global? Would you instead want to add sth. like the following?
```groovy
if (session.projects.contains(project)) {
	api.value('project', project.name)
}
```

Reply:
`buildScan.value('project', project.name)` is sufficient. 👍